src/redux/mainWallet/selectors/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+* Copyright 2017–2018, LaborX PTY
+* Licensed under the AGPL Version 3 license.
+*/
+
+import { createSelector } from 'reselect'
+import { DUCK_BITCOIN } from '@chronobank/bitcoin/redux/constants'
+import { DUCK_ETHEREUM } from '@chronobank/ethereum/redux/constants'
+
+const getBitDuck = (state) => state[DUCK_BITCOIN]
+const getEthDuck = (state) => state[DUCK_ETHEREUM]
+
+export const getSections = createSelector(
```

Comment:
Hmmm... Probably we need to create new packages/wallets scoped package and move this selector to this package.